pex/bin/pex.py

```diff
@@ -291,6 +291,17 @@ def configure_clp_pex_options(parser):
            'No value (alias for prefer, for backwards compatibility). '
            '[Default: %default]')
 
+  group.add_option(
+      '--include-pyc', '--no-include-pyc',
+      dest='include_pyc',
+      default=True,
+      action='callback',
+      callback=parse_bool,
+      help='Including .pyc files will result in slightly faster startup performance, but at the '
```

Comment:
I'm not sure if / how to include in help without writing a book, but this is only true for PEX sources, not anything added via requirements. So, forna requirements only pex, that's just the `__main__.py` and `.bootstrap/`.

Reply:
I don't follow how I should update for this. Do requirements ever have `.pyc` in them? It looks from a simple test of `pex six -o t.pex --no-compile; unzip t.pex -d t` that the six files is just the wheel itself and that there is no `.pyc` regardless of `--compile` or `--no-compile`.

Reply:
That's what I meant and I have no great suggestion. The basic worry is that a new PEX user sees --compile and expects using that option will get all bytecode compiled. It won't. For the perhaps typical non-Pants use case of requirements only, it will even dissapointingly compile almost nothing.